src/api/commands/setting/SettingSetCommand.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2017-2018, The Particl Market developers
+// Distributed under the GPL software license, see the accompanying
+// file COPYING or https://github.com/particl/particl-market/blob/develop/LICENSE
+
+import { inject, named } from 'inversify';
+import { validate, request } from '../../../core/api/Validate';
+import { Logger as LoggerType } from '../../../core/Logger';
+import { Types, Core, Targets } from '../../../constants';
+import { ProfileService } from '../../services/ProfileService';
+import { RpcRequest } from '../../requests/RpcRequest';
+import { Setting } from '../../models/Setting';
+import { RpcCommandInterface } from '../RpcCommandInterface';
+import { Commands } from '../CommandEnumType';
+import { BaseCommand } from '../BaseCommand';
+import { RpcCommandFactory } from '../../factories/RpcCommandFactory';
+import { SettingUpdateRequest } from '../../requests/SettingUpdateRequest';
+
+export class SettingSetCommand extends BaseCommand implements RpcCommandInterface<void> {
+
+    public log: LoggerType;
+
+    constructor(
+        @inject(Types.Core) @named(Core.Logger) public Logger: typeof LoggerType,
+        @inject(Types.Service) @named(Targets.Service.SettingService) private profileService: ProfileService
+    ) {
+        super(Commands.SETTING_SET);
+        this.log = new Logger(__filename);
+    }
+
+    /**
+     * data.params[]:
+     *  [0]: profile id
+     *  [1]: key
+     *  [2]: value
+     *
+     * @param data
+     * @param rpcCommandFactory
+     * @returns {Promise<void>}
+     */
+    @validate()
+    public async execute( @request(RpcRequest) data: RpcRequest, rpcCommandFactory: RpcCommandFactory): Promise<void> {
+        return await this.profileService.removeSetting({
+            profileId : data.params[0],
+            key : data.params[1],
+            value : data.params[2]
+        } as SettingUpdateRequest);
+    }
+
```

Comment:
SettingSetCommand and youre calling profileService.removeSetting?
Has this pr have tests which have been run or am I wasting my time?